### ISSUE TITLE
product details page not showing 

### ISSUE THREAD

**Issue by cayangaannika** [cayangaannika/ecommerce-pattern-library#3]
@thomasjbradley 

Hi Thomas,

I'm still having troubles linking my products page to the product details page. How can I share my pattern library with you to show the problem? 

Annika

**Comment by thomasjbradley** [comment]
You mean after your partner submitted the changes? Like you can’t see the version your partner created?

**Comment by cayangaannika** [comment]
Yes, my partner and I are having trouble linking my own product page to go to a md file product detail page. Even when I type in the md file name in the url it just says page not found. I'm not sure how to 
1. make my product detail page show up 
2. how to link it with a button on my card to go to the product detail(md file)page

@thomasjbradley
